server/server.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-console */
+const express = require('express');
+
+const app = express();
+const PORT = 5000;
+const mongoose = require('mongoose');
+const cookieParser = require('cookie-parser');
+const bodyParser = require('body-parser');
+require('dotenv').config();
```

Comment:
Are you using this anywhere? You can put the port number inside your env

Reply:
i was using it before, but after the decision of putting the exposing the username and password isnt a problem, i no longer am using env atm.